design-system-docs/src/_layouts/default.erb

```diff
@@ -4,7 +4,6 @@
     <meta charset="utf-8" />
     <%= seo %>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta name="robots" content="noindex, nofollow">
```

Comment:
Ah ok - is this why google was flagging the site as suspicious? 🤔 

Taking these out means Google will index the documentation site.

Reply:
I thought so initially (along with the fact that we have password fields examples). But as the issue hasn't happened for a while I think we can leave those as they are. That's also because we've been discussing having a different domain so probably worth allowing indexing once that's in place. I'll close the PR for now, but may re-open in the future